doc/flame/components.md

```diff
@@ -911,13 +911,41 @@ void main() {
 
 ## TiledComponent
```

Comment:
In the long run this section should be moved to `/doc/flame_tiled/`, just like with `flame_audio` for example.
But this doesn't have to be done in this PR if you don't want to.